client-examples/BrutalmazeClient.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Text;
+using System.Net;
+using System.Net.Sockets;
+
+namespace BrutalmazeClient
+{
+	class Program
+	{
+		private static Socket ConnectSocket(string host, int port)
+		{
+			IPHostEntry ipHostInfo = Dns.GetHostEntry(host);
+			IPAddress ipAddress = ipHostInfo.AddressList[0];
+			Socket tmp = new Socket(ipAddress.AddressFamily, SocketType.Stream, ProtocolType.Tcp);
+			tmp.Connect(ipAddress, port);
```

Comment:
This still causes *Connection refused* exception for me. Is there any problem with `tmp.Connect(host, port)` on your machine? Really socket is implement differently on Windows so if that doesn't work for you then you have to speak. We need more communication here bro.